mmengine/dataset/base_dataset.py

```diff
@@ -173,6 +173,8 @@ class BaseDataset(Dataset):
         pipeline (list, optional): Processing pipeline. Defaults to [].
         test_mode (bool, optional): ``test_mode=True`` means in test phase.
             Defaults to False.
+        backend_args (dict, optional): Arguments to instantiate the
+            corresponding backend. Defaults to None.
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.

Reply:
Makes sense. Fixed.